### PR TITLE
Update nginx.conf.j2

### PR DESCRIPTION
**Pull request by MauriceHendrix**
I have been experimenting with switching on gzip compression and it does make loading large graphs for stories much faster. I'm not 100% sure if this is the reight way to deply it @jonc125 ?

### CODE REVIEW

File: roles/geerlingguy.nginx/templates/nginx.conf.j2 [hunk 1]
```diff
@@ -47,7 +47,32 @@ http {
...
+        application/xml
+        application/xml+rss
+        application/xhtml+xml
+        application/x-font-ttf
+        application/x-font-opentype
+        image/svg+xml
+        image/x-icon;
+    gzip_buffers 16 8k;
+    gzip_min_length 512;
+{% endblock %}
```
Comment (jonc125) [inline roles/geerlingguy.nginx/templates/nginx.conf.j2:75]: You've doubled this line!

**Review by jonc125 — CHANGES_REQUESTED** [review]
This looks plausible, but I've not tested it in operation. How have you assessed it?

Normally I'd deploy on a local VM and check operation before deploying on scrambler.

**Comment by MauriceHendrix** [comment]
yup tried it on the cardiac server, seems ok & faster.

File: roles/geerlingguy.nginx/templates/nginx.conf.j2 (hunk 1 above)
Comment (jonc125) [reply roles/geerlingguy.nginx/templates/nginx.conf.j2:74]: It's the endblock line that appears twice, and you've now removed CSV from the list?

Comment (MauriceHendrix) [reply roles/geerlingguy.nginx/templates/nginx.conf.j2:75]: uups you're quite right ! should be better now

**Review by jonc125 — APPROVED** [review]

**Pull request merged**